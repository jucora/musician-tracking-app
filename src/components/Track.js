/* eslint no-console: ["error", { allow: ["warn", "error"] }] */

import React from 'react';
import Skill from './Skill';

const Track = (props) => {
  const { user } = props;

  return (
    <div>
      <h1>Track.it</h1>
      <h2>Welcome {user.email}</h2>
      <Skill />
    </div>
  );
};

export default Track;
