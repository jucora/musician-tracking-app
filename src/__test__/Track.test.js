import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import Track from '../components/Track';

afterEach(cleanup);

it('renders', () => {
  const { asFragment } = render(
    <Track user={{}} loggedInStatus="LOGGED_IN" history={{ push: () => {} }} />
  );
  expect(asFragment()).toMatchSnapshot();
});
