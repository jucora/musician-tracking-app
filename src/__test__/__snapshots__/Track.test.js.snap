// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="track"
  >
    <h2
      class="trackTitle"
    >
      Hello
    </h2>
    <div
      class="totalScore"
    >
      <div>
        <h2>
          Total Hours
        </h2>
        <p>
          0
        </p>
      </div>
    </div>
    <div
      class="skillContainer"
    />
  </div>
</DocumentFragment>
`;
